Add correction about TCP_TLS routers in docs (#10489)
traefik/traefik: docs/content/routing/routers/index.md

### General

> In case of non-TLS Entrypoint, both HTTP routers and TCP routers listen to the same EntryPoint, the TCP routers will apply *before* the HTTP routers.

Reviewer: ```suggestion
For non-TLS connections, if HTTP and TCP routers listen on the same EntryPoint, the TCP routers will apply *before* the HTTP routers.
```

> In case of TLS Entrpoint, both HTTPS and TCP-TLS routers listen to the same EntryPoint, the HTTPS routers will apply *before* the TCP-TLS routers.

Reviewer: ```suggestion
For TLS connections, if HTTPS and TCP-TLS routers listen on the same EntryPoint, the HTTPS routers will apply *before* the TCP-TLS routers.
```
